.travis.yml

```diff
@@ -105,6 +105,38 @@ jobs:
 
     - stage: Test
       php: 7.1
+      env: DB=mariadb MARIADB_VERSION=10.2
+      addons:
+        mariadb: 10.2
+    - stage: Test
+      php: 7.2
+      env: DB=mariadb MARIADB_VERSION=10.2
+      addons:
+        mariadb: 10.2
+    - stage: Test
+      php: nightly
+      env: DB=mariadb MARIADB_VERSION=10.2
+      addons:
+        mariadb: 10.2
+
+    - stage: Test
+      php: 7.1
+      env: DB=mariadb.mysqli MARIADB_VERSION=10.2
```

Comment:
This sort of breaks the purpose of the `$DB` variable - it's DB + driver now. Maybe adding 2nd optional variable `$DRIVER` would be better in this case.

Reply:
Good, what name for the 2nd variable ?

`MARIADB_DRIVER' ?

Do you know where to set it as default (I assume not under env: MARIADB_DRIVER=mysql ? Not sure with travis jobs) ?

Reply:
Simply `$DRIVER` would be enough. But I see there's already `DB=mysqli` which is not really a DB. Feel free to leave it as is it for now, we can handle this after this PR is merged. 😉

Reply:
I leave it like this then ;) @Majkl578 I'm about to drop some parts of this P/R, just detailed 3 questions in the last comment. Would be wonderful to have your advices on this.